packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-global/instance-at-proposals-corejs-3.8/output.mjs

```diff
@@ -27,6 +27,7 @@ import "core-js/modules/es.typed-array.to-locale-string.js";
 import "core-js/modules/es.typed-array.to-string.js";
 import "core-js/modules/esnext.typed-array.at.js";
 import "core-js/modules/es.array.at.js";
+import "core-js/modules/es.string.at-alternative.js";
```

Comment:
This is a bug, `es.string.at-alternative` should be in "3.17" in https://unpkg.com/browse/core-js-compat@3.17.0/modules-by-versions.json